source/scss/modules/_header.scss

```diff
@@ -1,41 +1,41 @@
 .header {
   text-align: center;
+
+  a {
+    position: relative;
+    margin: 0 auto;
+    background: $c--ice-cold;
+    @include rem( border-radius, .1875 );
+    color: $c--blue-whale;
+    display: block;
+    width: 50%;
+    font-family: $secondary-font;
+    font-size: 1.25em;
+    font-weight: 700;
+    line-height: 1;
+    @include rem( padding, .35, .5 );
+    text-transform: uppercase;
+    transition: filter .25s ease-in-out;
+    @include sb-media ( 45em ) {
+      display: inline-block;
+      width: auto;
+      bottom: 90%;
+      left: -20%; 
+    }
+    &:hover, &:active, &:visited {
+      filter: brightness(1.1);
```

Comment:
I think a good hovering effect here could be hovering to the original color of button. This would just be an inverted version of what it was before
![Screen Shot 2022-02-04 at 5 24 02 PM](https://user-images.githubusercontent.com/34315878/152611291-0b7be5e0-22d3-4615-adf6-f373e42f452c.png)

Reply:
I like this hover effect much better! I changed it to change to the c--juniper-light color created for the footer content to make sure the contrast of the hovered button still met WCAG requirements.